sys/include/net/wot/coap.h

```diff
@@ -4,28 +4,13 @@
 #include "net/wot.h"
 #include "net/gcoap.h"
 
-typedef struct {
+typedef struct wot_td_coap_form {
     const coap_resource_t *coap_resource;
     wot_td_form_t *form;
```

Comment:
Thank you for these changes! While implementing the new script version I noticed that there is only a `form` reference in this struct. In the affordances and at the thing level, however, we can have a linked list of forms that could also point to different resources. If this is already reflected in the data structure could you maybe provide a example how the respective references have to be defined? (Sorry that this is such a hassle :/)

As I mentioned in the issue, I think it might actually be easier if you register the linked list of CoAP resources separately and then point from the `wot_td_form_t` to the resources instead of the other way around.